pages/en/lb4/Getting-started.md

```diff
@@ -5,35 +5,202 @@ keywords: LoopBack 4.0, LoopBack 4
 tags:
 sidebar: lb4_sidebar
 permalink: /doc/en/lb4/Getting-started.html
-summary:
+summary: Write and run a LoopBack 4 "Hello World" project
 ---
-## Prerequisites: JavaScript
+## Prerequisites
 
-Like other modern JavaScript frameworks (for example, Angular 2 and React) LoopBack 4 has an opinion about what flavor of JavaScript to use in your application.
-The starting point is [ECMAScript 6](http://www.ecma-international.org/ecma-262/6.0/), plus the following non-standard add-ons:
+Follow the instructions in [Installation](Installation.html) to install the prerequisites for a LoopBack 4 project.
 
- - Decorators
- - `async` / `await`
- - ES2015-modules
+With LoopBack 4 you can code in JavaScript or TypeScript.  Regardless of which you use, you'll need to do the following:
 
-**All examples and snippets in the documentation use this flavor of JavaScript.**
+ 1. Create a new directory called `lb4-hello-world` and make that your current directory:
+ ```
+ $ mkdir lb4-hello-world
+ $ cd lb4-hello-world
+ ```
+ 1. Create a `package.json` by following the instructions below.
```

Comment:
there are 2 `1.`  line 16 and line 21.

Reply:
In markdown, an ordered list items can be denoted by 
```
1. One
1. Two
1. Three
```
And it is rendered as 

1. One
1. Two
1. Three

So, what's there is correct.